homework5/tests/task2_test.py

```diff
@@ -0,0 +1,31 @@
+from unittest.mock import Mock
+
+from homework5.save_original_info import print_result
+
+
+def set_custom_sum_mock():
+    custom_sum = Mock(return_value=10)
+    custom_sum.__doc__ = """Docstring"""
+    custom_sum.__name__ = "custom_sum"
+    return custom_sum
+
+
+def test_custom_sum_doc():
+    custom_sum = set_custom_sum_mock()
+    decorated_custom_sum = print_result(custom_sum)
+
+    assert decorated_custom_sum.__doc__ == custom_sum.__doc__
+
+
+def test_custom_sum_name():
+    custom_sum = set_custom_sum_mock()
+    decorated_custom_sum = print_result(custom_sum)
+
+    assert decorated_custom_sum.__name__ == custom_sum.__name__ == "custom_sum"
+
+
+def test_custom_sum_original_func():
+    custom_sum = set_custom_sum_mock()
+    decorated_custom_sum = print_result(custom_sum)
+
+    assert decorated_custom_sum.__original_func() == custom_sum() == 10
```

Comment:
You should check if the `print_result` decorator actually print something to stdout.